packages/MarkdownEditor-Tests.package/MarkdownHeadingsTest.class/instance/testMatchHeadingLevels1To6WithText.st

```diff
@@ -1,6 +1,5 @@
 tests - matching
 testMatchHeadingLevels1To6WithText
-	
-	#('# MiniMetro' '## the fastest game' '### on the planet!' '#### cool play style' 
-	'##### endless fun' '###### universal gratitude of passengers') do: [:string |
+
+	{'# H1'. '## H2'. '### H3'. '#### H4'. '##### H5'. '###### H6'} do: [:string |
```

Comment:
Let us have a little fun with test strings... This one really didn't mess with understandability...

Reply:
I like it like that. Especially that it fits in one line.